website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -171,9 +186,13 @@ This interprets command-line arguments that follow the format:
 --<option>=<value>
 ```
 
-> **Note:** Appending the method `.withValidation` will check for required
+<span class="language-java">
+
+> Appending the method `.withValidation` will check for required
 > command-line arguments and validate argument values.
 
+</span>
+
 Building your `PipelineOptions` this way lets you specify any of the options as
```

Comment:
Does highlighting `PipelineOptions` in the guide for go developers, refer specifically to the var [`PipelineOptions`](https://github.com/apache/beam/blob/master/sdks/go/pkg/beam/forward.go#L157).  Should it not be mentioned for go developers reading this guide, particularly new developers?  I can see why it was included for the Java documentation.

Reply:
Should there be:

```
<span class="language-java language-py">
Building your `PipelineOptions` this way lets you specify any of the options as a command-line argument.
</span>
<span class="language-go">
Defining flag variables this way lets you specify any of the options as a command-line argument.
</span>
```

Reply:
Done.